### ISSUE TITLE
Schedule not running

### ISSUE THREAD

**Issue by ismijgelijk** [fluentscheduler/FluentScheduler#84]
I'm not sure wether this is a bug, or me using the library incorrectly again, but I don't really know how or  where else I can ask.

I was told in issue 74 that I was using the library incorrectly then, so I changed my code to the following:

```
Timespan timeSpan = Settings.TaskTimeOfDay;
bool runOnceOnStartup = Settings.TaskRunOnceOnStartup;
 if (runOnceOnStartup)
 {
     Settings.MarkActivitiesAsCompleteTaskRunOnceOnStartup = false;
     Schedule<MarkActivitiesAsCompleteTask>().ToRunNow();
 }
 Schedule<MarkActivitiesAsCompleteTask>.ToRunEvery(1).Days().At(timeSpan.Hours, timeSpan.Minutes);
```

The runOnceOnStartup works as intended.
The run every day part however does not.
My task does not get executed, and when I look at the NextRunTime-property, its value is {1/01/0001 0:00:00}

The version I have been using so far was 3.1.46.
I just tried with version 4.1.1.0, and the date is still the same.
I haven't had the chance to run for a full day.

Could you tell me what is going wrong here?
Thank you

**Comment by tallesl** [comment]
That's odd, your code seems OK.

As for the NextRun being `1/01/0001 0:00:00`, you probably inspected before the `JobManager.Initialize` call. `Schedule` only sets `CalculateNextRun`, the actual `NextRun` is set by the `JobManager`.

Can you double check your job class and hour and minute being passed there?

**Comment by ismijgelijk** [comment]
Hello

As far as I can tell, the jobmanager is initialized.
In my code, it's actually initialized more than once.

Either way, I just tried to make a simplified version of my project, and it seems there, everything works as expected.
I also managed to work around the issue for now by calling Torunonce().andevery() instead of ToRunEvery()

I'll see if I can reproduce the issue later, and let you know, if you're still interested :)

Anyway, thank you already for being willing to look into it, it's much appreciated
